Description: Fix Python 3.4 compatibility
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-05-28

--- python-wrapt-1.8.0.orig/tests/test_object_proxy.py
+++ python-wrapt-1.8.0/tests/test_object_proxy.py
@@ -1473,7 +1473,7 @@ class SpecialMethods(unittest.TestCase):
 
         proxy = wrapt.ObjectProxy(instance)
 
-        self.assertEqual(format(instance, 's'), format(proxy, 's'))
+        self.assertEqual(format(instance, ''), format(proxy, ''))
 
     def test_list_reversed(self):
         instance = [1, 2]
