Description: Do not import embedded six.py in tests
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-05-28

--- python-wrapt-1.8.0.orig/tests/test_function_wrapper.py
+++ python-wrapt-1.8.0/tests/test_function_wrapper.py
@@ -5,7 +5,7 @@ import unittest
 import wrapt
 import wrapt.wrappers
 
-from wrapt import six
+import six
 
 class TestClassInheritence(unittest.TestCase):
 
--- python-wrapt-1.8.0.orig/tests/test_instancemethod.py
+++ python-wrapt-1.8.0/tests/test_instancemethod.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_function.py
+++ python-wrapt-1.8.0/tests/test_function.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_outer_classmethod.py
+++ python-wrapt-1.8.0/tests/test_outer_classmethod.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_nested_function.py
+++ python-wrapt-1.8.0/tests/test_nested_function.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_inner_classmethod.py
+++ python-wrapt-1.8.0/tests/test_inner_classmethod.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_adapter.py
+++ python-wrapt-1.8.0/tests/test_adapter.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_adapter_py33.py
+++ python-wrapt-1.8.0/tests/test_adapter_py33.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_outer_staticmethod.py
+++ python-wrapt-1.8.0/tests/test_outer_staticmethod.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_inner_staticmethod.py
+++ python-wrapt-1.8.0/tests/test_inner_staticmethod.py
@@ -6,7 +6,7 @@ import imp
 
 import wrapt
 
-from wrapt import six
+import six
 
 DECORATORS_CODE = """
 import wrapt
--- python-wrapt-1.8.0.orig/tests/test_update_attributes.py
+++ python-wrapt-1.8.0/tests/test_update_attributes.py
@@ -4,7 +4,7 @@ import unittest
 
 import wrapt
 
-from wrapt import six
+import six
 
 @wrapt.decorator
 def passthru_decorator(wrapped, instance, args, kwargs):
--- python-wrapt-1.8.0.orig/tests/test_object_proxy.py
+++ python-wrapt-1.8.0/tests/test_object_proxy.py
@@ -9,7 +9,7 @@ is_pypy = '__pypy__' in sys.builtin_modu
 
 import wrapt
 
-from wrapt import six
+import six
 
 OBJECTS_CODE = """
 class TargetBaseClass(object):
